src/tools/wasm-reduce.cpp

```diff
@@ -429,6 +429,25 @@ struct Reducer : public WalkerPass<PostWalker<Reducer, UnifiedExpressionVisitor<
       handleCondition(select->condition);
     } else if (auto* sw = curr->dynCast<Switch>()) {
       handleCondition(sw->condition);
+      // Try to replace switch targets with the default
+      for (auto& target : sw->targets) {
+        if (target != sw->default_) {
+          auto old = target;
+          target = sw->default_;
+          if (!tryToReplaceCurrent(curr)) {
+            target = old;
+          }
+        }
+      }
+      // Try to shorten the list of targets.
+      while (sw->targets.size() > 1) {
+        auto last = sw->targets.back();
```

Comment:
can this just be `pop_back()`?

Reply:
Good idea, fixed. Found a few more in this file too.

(Honestly beceause `pop_back()` doesn't return a value I think I just gave up and forgot about it...)